.github/release-drafter.yml

```diff
@@ -1,19 +1,19 @@
-name-template: v$NEXT_PATCH_VERSION
-tag-template: v$NEXT_PATCH_VERSION
+name-template: 'v$NEXT_PATCH_VERSION'
+tag-template: 'v$NEXT_PATCH_VERSION'
 categories:
-  - title: 💡 New Features
-    label: feature
-  - title: 🔨 Breaking
-    label: breaking
-  - title: 🚀 Efficiency Improvement
-    label: efficiency
-  - title: 🐛 Bug Fixes
-    label: fix
-  - title: 📖 Documentation
-    label: doc
-  - title: 🧰 Maintenance
-    label: maintenance
-tag-template: - $TITLE @$AUTHOR (#$NUMBER)
+  - title: '💡 New Features'
+    label: 'feature'
+  - title: '🔨 Breaking'
+    label: 'breaking'
+  - title: '🚀 Efficiency Improvement'
+    label: 'efficiency'
+  - title: '🐛 Bug Fixes'
+    label: 'fix'
+  - title: '📖 Documentation'
+    label: 'doc'
+  - title: '🧰 Maintenance'
+    label: 'maintenance'
+change-template: '- TITLE @TITLE @AUTHOR (#$NUMBER)'
```

Comment:
Why `TITLE` is duplicated here?

I think we can follow this template which looks quite clear 
https://github.com/DefectDojo/django-DefectDojo/blob/9986e86bc6fc84c79e96daf515e26add79a74652/.github/release-drafter.yml#L24